src/components/MessageListItem.react.js

```diff
@@ -61,24 +65,33 @@ class MessageListItem extends React.Component {
         let lat, lng;
 
         let SpecialResponseChoice = data.answers[0].actions[0];
-        if (SpecialResponseChoice.type === "table") {
+        if (SpecialResponseChoice.type === 'table') {
             let coloumns = data.answers[0].actions[0].columns;
             let tableheader = Object.keys(coloumns).map(key =>{
-              return(<TableHeaderColumn>{coloumns[key]}</TableHeaderColumn>);
+              return <TableHeaderColumn key={key}>
+                {coloumns[key]}
+              </TableHeaderColumn>;
             });
-            let rows = data.answers[0].data.map(eachrow => {
+
+            let rows = data.answers[0].data.map((eachrow, i) => {
               let rowcols = Object.keys(coloumns).map(key =>{
-                return(<TableRowColumn><Linkify properties={{target:"_blank"}}>{eachrow[key]}</Linkify></TableRowColumn>);
+                return <TableRowColumn key={key}>
+                  <Linkify properties={{ target: '_blank' }}>
+                    {eachrow[key]}
+                  </Linkify>
+                </TableRowColumn>;
               });
               return(
-                <TableRow>{rowcols}</TableRow>
+                <TableRow key={i}>{rowcols}</TableRow>
```

Comment:
Again, I have done only the changes that the eslint config required, so our existing codebase was having issues with linting, and I have resolved those just to pass the tests.

Otherwise, I'd have to fix it first, create a PR and then add another rule to eslint. I thought of adding this change to initial eslint PR.